index.js

```diff
@@ -2,27 +2,61 @@
 const semver = require('semver');
 const execa = require('execa');
 const del = require('del');
-const chalk = require('chalk');
+const Listr = require('listr');
+const split = require('split');
+require('any-observable/register/rxjs-all');
+const Observable = require('any-observable');
+const streamToObservable = require('stream-to-observable');
 
 const VERSIONS = ['major', 'minor', 'patch', 'premajor', 'preminor', 'prepatch', 'prerelease'];
 
-const log = str => console.log(chalk.bold(`\n${chalk.cyan('›')} ${str}\n`));
-
 const exec = (cmd, args) => {
-	// TODO Switch to `{stdio: 'inherit'}` instead of manual logging when a new execa version is released
-	const result = execa.sync(cmd, args);
+	// Use `Observable` support if merged https://github.com/sindresorhus/execa/pull/26
+	const cp = execa(cmd, args);
 
-	if (result.stdout) {
-		console.log(result.stdout);
-	}
+	return Observable.merge(
+		streamToObservable(cp.stdout.pipe(split())),
+		streamToObservable(cp.stderr.pipe(split()))
+	).filter(Boolean);
```

Comment:
Will the observable reject if `cp` rejects? Not sure about this being the same as my previous attempt :).